Pipfile.lock

```diff
@@ -181,9 +181,9 @@
         },
         "holidays": {
```

Comment:
This change breaks our environment. Prophet does not support holidays>=0.9.12

Reply:
Maybe a good idea to give a version specified in the Pipfile

Reply:
Ah you did it! :)